supremm/pcparchive.py

```diff
@@ -62,14 +65,29 @@ def get_datetime_from_pmResult(result):
     """
     return get_datetime_from_timeval(result.contents.timestamp)
 
-def extract_and_merge_logs(job, conf, resconf):
+def extract_and_merge_logs(job, conf, resconf, opts):
```

Comment:
OK, this should be all set. I added the full functionality while I was doing it.